builtin/mainmenu/content/contentdb.lua

```diff
@@ -169,15 +169,16 @@ function contentdb.get_package_by_id(id)
 	return contentdb.package_by_id[id]
 end
 
-
-function contentdb.calculate_package_id(type, author, name)
-	local id = author:lower() .. "/"
+local function strip_game_suffix(type, name)
```

Comment:
```suggestion

local function strip_game_suffix(type, name)
```

(readd removed spacing)